validate/validate.proto

```diff
@@ -553,9 +553,23 @@ message StringRules {
         // Uuid specifies that the field must be a valid UUID as defined by
         // RFC 4122
         bool uuid     = 22;
+
+        // WellKnownRegex specifies a common well known pattern defined as a regex.
+        KnownRegex well_known_regex = 24;
     }
 }
 
+// WellKnownRegex contain some well-known patterns.
+enum KnownRegex {
+  UNKNOWN = 0;
+
+  // HTTP header name as defined by RFC 7230.
+  HTTP_HEADER_NAME = 1;
```

Comment:
I'm tempted to suggest `RFC_7230_HEADER_NAME` here, but it's completely unreadable in annotations (but is that tiny bit more precise).

Reply:
Unless there are alternative specs for HTTP headers that I don't know about, I'd suggest leaving the existing name for the sake of readability at the usage sites. The comment here is a sufficiently precise pointer to the actual RFC spec.